qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/sections/BaseWhenSection.java

```diff
@@ -30,6 +33,35 @@ public abstract class BaseWhenSection extends Section {
 
 	private static final String VALUE = "value";
 
+	private static final Map<String, Operator> singleParamOperators;

```

Comment:
To avoid using 2 Map, I will create a new class CaseOperator which extends Operator and which provide a new field isMulti:

```java
package com.redhat.qute.parser.template;

public class CaseOperator extends Operator {

	private final boolean multi;
	
	public CaseOperator(String name, boolean multi, String[] aliases) {
		super(name, aliases);
		this.multi = multi;
	}

	public boolean isMulti() {
		return multi;
	}
}

```

Replace 

`private static final Map<String, Operator> singleParamOperators;
`
with 

`private static final Map<String, Operator> caseOperators;`